content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -2,6 +2,1344 @@
+
+
+## Quick Start
+
+{{< tabs name="Python runtime setup" >}}
+{{% tab name="Windows" %}}
+
+For Windows, open a command prompt. Then, run the O3DE Python runtime using one of the following ways, and enter the command you want to perform. Note: Replace `<engine>` with the path to your engine. 
+
+- Launch python and run the `o3de.py` script.
+
+    ```cmd
+    <engine>/python/python.cmd <engine>/scripts/o3de.py <command>
+    ```
+
+- Run the `o3de.bat` batch file directly.
+
+    ```cmd
+    <engine>/scripts/o3de.bat <command>
+    ```
+
+**Example**
+
+For example, to register the engine, you can enter the following command. 
+
+```cmd
+<engine>/scripts/o3de.bat register --this-engine
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+For Linux, open a terminal. Then, run the O3DE Python runtime using one of the following ways, and enter the command you want to perform. Note: Replace `<engine>` with the path to your engine. 
+
+- Launch python and run the `o3de.py` script.
+
+    ```bash
+    <engine>/python/python.sh <engine/scripts/o3de.py
+    ```
+
+- Run the `o3de.sh` shell script file directly.
+
+    ```bash
+    <engine>/scripts/o3de.sh
+    ```
+
+**Example**
+
+For example, to register the engine, you can enter the following command. 
+
+```bash
+<engine>/scripts/o3de.sh register --this-engine
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## Commands
+
+
+| Command | Description | 
+| - | - |
+| [`get-global-project`](#get-global-project) | Gets the global project that is registered to the engine. |
+| [`set-global-project`](#set-global-project) | Sets the specified project as the engine's global project. |
+| [`create-template`](#create-template) | Creates a template out of the specified source path. |
+| [`create-from-template`](#create-from-template) | Creates a template from an existing template.  |
+| [`create-project`](#create-project) | Creates a new project. |
+| [`create-gem`](#create-gem) | Creates a new Gem. |
+| [`register`](#register) | Register an O3DE object. |
+| [`register-show`](#register-show) | Shows the registered O3DE objects. |
+| [`get-registered`](#get-registered) | Shows the registered O3DE object's path that is mapped to the specified name. |
+| [`enable-gem`](#enable-gem) | Enables the Gem in your project, so you can use the assets or code the Gem provides. |
+| [`disable-gem`](#disable-gem) | Disables the Gem in your project. |
+| [`sha256`](#sha256) | Creates a secure hash algorithm (SHA-256). |
+<!-- | [`edit-project-properties`](#edit-project-properties) |  | -->
+
+
+
+## `get-global-project`
+
+Gets the global project that is registered to the engine in the specified `.setreg` file at `<user>/.o3de/Registry/`.
+
+
+```cmd
+usage: o3de.py get-global-project [-h] [-i INPUT_PATH]
+```
+
+### Usage
+
+```cmd
+o3de.bat get-global-project -i SETREG_PATH
```

Comment:
If you intend this to be a usage _example_, it needs an explanation of the scenario. Perhaps use a comment in the block? Also, SETREG_PATH doesn't really add anything over INPUT_PATH that's shown in the formatting above.

This particular command might also benefit from two examples.
```suggestion
// Read the global project from <USER_PATH>/.o3de/Registry/bootstrap.setreg.
o3de.bat get-global-project

// Read the global project from a specified path.
o3de.bat get-global-project -i C:\users\<USERNAME>\.o3de\Registry\my-custom.setreg
```

Reply:
I guess the downside to using actual paths in every usage example is that it can quickly lengthen these examples, enough to make it harder to process all the information shown.

I propose that the guideline here should be to use it when there's added value in showing an actual example over just a variable. For example, there's implicit value in showing the path to the setreg file in the get-global-project's `-i` parameter, and the set-global-project's `-o` parameter because users should typically put these files in the .o3de/Registry directory and the filename extension should be `.setreg`.